### DISCUSSION TITLE
How to add routes or remove routes in axum's handler？

### DISCUSSION THREAD

**Discussion by cnwzhu** [tokio-rs/axum#2055] [Q&A]
I want to modify the route in the handler, how to implement it in axum?

**Comment by davidpdrsn** [comment]
First I'd like to ask why you need this?

**Reply by cnwzhu** [reply]
I want to implement a cms system, and I want it to be very scalable, so I want to implement it by means of plugins. So I want to be able to modify the route when installing the plugin
